mingw-w64-cv2pdb/PKGBUILD

```diff
@@ -3,7 +3,7 @@
 _realname=cv2pdb
 pkgbase="mingw-w64-${_realname}"
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=0.50
+pkgver=0.51.8.g9286b93
```

Comment:
I would prefer to wait for a full release than to upgrade to an interim version.